### ISSUE TITLE
[Feature Request] adding 'ld+json' schema to allow a faster search in apkmirror

### ISSUE THREAD

**Issue by eladkarako** [archon810/apkmirror-public#18]
If you'll embedded the following code into your index-page,

```html
<script type="application/ld+json">
{
  "@context":           "http://schema.org"
, "@type":              "WebSite"
, "url":                "https://www.apkmirror.com/"
, "name":               "APKMirror - Free APK Downloads"
, "alternateName":      "Download Free Android APKs #APKPLZ"
, "logo":               "https://www.apkmirror.com/wp-content/themes/APKMirror/images/apple-touch-icon-180x180.png"
, "potentialAction": {  "@type": "SearchAction"
                     ,  "target": "https://www.apkmirror.com/?post_type=app_release&searchtype=apk&s={search_term_string}"
                     ,  "query-input": "required name=search_term_string"
                     }
}
</script>
```

minified: <code>&lt;script type="application/ld+json"&gt;{"@context":"http://schema.org","@type":"WebSite","url":"https://www.apkmirror.com/","name":"APKMirror - Free APK Downloads","alternateName":"Download Free Android APKs #APKPLZ","logo":"https://www.apkmirror.com/wp-content/themes/APKMirror/images/apple-touch-icon-180x180.png","potentialAction":{"@type":"SearchAction","target":"https://www.apkmirror.com/?post_type=app_release&searchtype=apk&s={search_term_string}","query-input":"required name=search_term_string"}}&lt;/script&gt;</code>

Chrome (and FF too, think..) will allow users to search the website more easily,
by embedding the website search engine into the browser's address-bar.

You can tweak it (and add more stuff), but this the minimal markup needed.

It also provides few "SEO points" with Google search for "friendly website" (I think..)

I've wrote a small section about it on <a href="http://icompile.eladkarako.com/apkmirror-search/">my website</a> <sub><em> (no self promotion is intended)</em></sub>
which was more for me, since I've kept loosing the search bar/button when I've browsed your website within my mobile phone.

Hope you'll have it :]

thanks!

**Comment by archon810** [comment]
Thanks for the suggestion.

Additional info:
http://schema.org/SearchAction.
https://developers.google.com/search/docs/data-types/sitelinks-searchbox

**Comment by archon810** [comment]
@eladkarako Turns out we already had this ld+json stuff, including SearchAction all along (it was added by an SEO plugin). Google maybe chooses to ignore it and doesn't add a search bar in search results for some reason (maybe they need to enable that manually per-site). And the Chrome site search has worked for me all along too.

I just made a few little tweaks, but yeah, the code you wanted is already there.
